pkg/cri/sbserver/service.go

```diff
@@ -350,7 +350,7 @@ func (c *criService) register(s *grpc.Server) error {
 // imageFSPath returns containerd image filesystem path.
 // Note that if containerd changes directory layout, we also needs to change this.
 func imageFSPath(rootDir, snapshotter string) string {
-	return filepath.Join(rootDir, plugin.SnapshotPlugin.String()+"."+snapshotter)
+	return filepath.Join(rootDir, plugins.SnapshotPlugin.String()+"."+snapshotter)
```

Comment:
Maybe unrelated to this PR. But this can point to non-existing directory if the user uses remote snapshotter that might not have the rootdir managed by containerd daemon.

Reply:
There are many things wrong with this code and the approach of using a static path for determining relevant filesystem info. Unrelated to this PR but something to be figured out. There is an effort within signode to define a better stats API including for filesystem data. This is especially relevant as we try to support multiple snapshotters within CRI.

Reply:
This was pretty good timing 😆 @ktock just pointed me here, but we've been seeing a kubelet error related to this for a couple months now apparently. https://github.com/containerd/containerd/pull/9216. That change makes it so snapshotters can return whatever the rootpath is (especially useful for remote sns).